recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
@bgruening another issue: if you look at [this recipe](https://github.com/bioconda/bioconda-recipes/blob/14f00b12ccdc27bb42379f2f3bb0d860deb20c1d/recipes/krona/2.5/build.sh), it installs the krona source into $PREFIX/krona. This will actually persist in the installed package. If that is intended, we should at least stick to the FHS and put it into $PREFIX/share/krona or even $PREFIX/src/krona.
@dmaticzka sorry to hijack your PR, once the proper copying to bin is there again, I will be happy to merge!